src/renderer/components/ConfirmationModal.tsx

```diff
@@ -14,6 +14,9 @@ export interface ConfirmationModalProps {
   onSubmit: () => void;
   title: string;
   confirmText?: string;
+  cancelText?: string;
+  cancelColor?: "secondary" | "inherit" | "default" | "primary" | undefined;
```

Comment:
This doesn't need to be both optional and undefined.

```suggestion
  cancelColor?: "secondary" | "inherit" | "default" | "primary";
```